HW_3/task_2.py

```diff
@@ -0,0 +1,15 @@
+def get_duplicates(lst):
+    duplicates = []
+    for item in lst:
+        if lst.count(item) > 1 and item not in duplicates:
```

Comment:
Чтобы не добавлять одно условие, можно сделать так.
```suggestion
    for item in set(lst):
        if lst.count(item) > 1:
```